cpu/samd5x/periph/rtc.c

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (C) 2014 Baptiste CLENET
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_saml21
```

Comment:
Hmm. This file is an unmodified copy of `cpu/saml1x/periph/rtc.c`, 236 lines. How are we going to maintain that?

Reply:
#11317 and #11318 will resolve this.